src/primitives.coffee

```diff
@@ -27,6 +27,61 @@ class ElementType
 
   set: (@content) -> @
 
+  getProperty: -> new ErrorType 'Element does not have method getProperty', @
+
+  has: -> false
+
+  map: -> new ErrorType 'Element does not have method map', @
```

Comment:
How about a `_noElement` function so you can do:

```coffeescript
map: -> _noElement('map')
```
I think that would be a lot DRYer. Really need method missing functionality for this.